exercises/K-functions-parameters/exercise2.js

```diff
@@ -1,4 +1,6 @@
-// Declare your function first
+function divide(a, b) {
+    return 3 / 4;
```

Comment:
The function should divide the two parameters that it has been passed (they're passed in as `a` and `b`)